crates/neon/src/types_impl/buffer/mod.rs

```diff
@@ -232,7 +232,11 @@ pub struct Region<'cx, T: Binary> {
     phantom: PhantomData<T>,
 }
 
-impl<'cx, T: Binary> Region<'cx, T> {
+impl<'cx, T> Region<'cx, T>
+where
+    T: Binary,
+    JsTypedArray<T>: Value,
```

Comment:
Just curious, why did this end up needing this extra `JsTypedArray<T>: Value` constraint but didn't need it before?

Reply:
Removal of the `Managed` trait meant that in a couple of places (here included), we no longer had a blanket impl of `Value` for all `T`. Same problem here: https://github.com/neon-bindings/neon/pull/970/files#diff-62376b9ae514f1c89cc0387cf246aec628cea758623a00d3b41d4e4381828791R648